common/src/main/java/tachyon/Constants.java

```diff
@@ -145,6 +145,7 @@
   public static final String MASTER_WEB_PORT = "tachyon.master.web.port";
   public static final String MASTER_TEMPORARY_FOLDER = "tachyon.master.temporary.folder";
   public static final String MASTER_HEARTBEAT_INTERVAL_MS = "tachyon.master.heartbeat.interval.ms";
+  public static final String MASTER_TTL_INTERVAL_MS = "tachyon.master.ttl.interval.ms";
```

Comment:
NIT: why having an word "interval" in the property name? I don't think TTL describes an interval concept.